portaljs-pages/page-admin-case-list/resources/page.properties

```diff
@@ -0,0 +1,5 @@
+displayName=Bonita Admin Case List
+name=custompage_adminCaseList
+description=Default admin case list
+resources=[GET|bpm/case, DELETE|bpm/case, GET|bpm/process, GET|bpm/archivedCase, DELETE|bpm/archivedCase, GET|system/session, GET|system/feature, GET|system/i18ntranslation]
```

Comment:
As GET|system/i18ntranslation is not filtered by the REST API authorization filter (it can be accessed by anyone), it should be removed from this list

Reply:
OK, I will probably do the same thing for the process details then

Reply:
:+1:

Reply:
Done